packages/vuetify/src/components/VTimeline/VTimeline.tsx

```diff
@@ -18,12 +15,19 @@ import { VTimelineSymbol } from './shared'
 import type { Prop } from 'vue'
 
 export type TimelineDirection = 'vertical' | 'horizontal'
-export type TimelineSide = 'before' | 'after' | undefined
+export type TimelineSide = 'start' | 'end' | undefined
+export type TimelineAlign = 'center' | 'start'
```

Comment:
The examples are using `top`. If this is vertical only then stick to top/center/bottom instead of logical names.

Reply:
It is not vertical only. It works in horizontal mode as well